test/onnx/test_fx_op_consistency.py

```diff
@@ -275,6 +275,10 @@
             "Chunk", "uint8, int8, int16, float16"
         ),
     ),
+    xfail(
+        "chunk", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES + onnx_test_common.FLOAT_TYPES,
```

Comment:
```suggestion
        "chunk",
```

fine to not include dtypes if it's everything